app/javascript/App/bundles/common/components/CoinCharts/TradingViewChart/index.js

```diff
@@ -1,10 +1,34 @@
 import React, { Component } from 'react'
 import _ from 'lodash'
 import Datafeed from './Datafeed'
+import { withStyles, createStyles } from '@material-ui/core/styles'
 
 const containerID = 'tradingview'
 
-export default class TradingViewChart extends Component {
+/***
+ * Darkmode is implemented by using filters to achieve a reasonable background colour/colour scheme
```

Comment:
YES YES YES.

This is the easy way to implement dark mode, definitely approve.

Reply:
Thanks for the comment as well, super helpful!